scripts/o3de/o3de/repo_properties.py

```diff
@@ -90,7 +90,10 @@ def create_remote_object_archive(src_data_path: pathlib.Path,
         FileNotFoundError: If the json_data_path does not exist.
         ValueError: If the json_data_path is not a dict.
     """
-
+    if download_prefix is None:
+        logger.error('The -download-prefix must be provided. A url prefix for a file attached to a Github release might look like this:'
```

Comment:
The argument is missing a '-'
```suggestion
        logger.error('The --download-prefix argument must be provided. A url prefix for a file attached to a Github release might look like this:'
```